invenio_rdm_records/theme/templates/invenio_rdm_records/details/doi.html

```diff
@@ -9,6 +9,7 @@
 {%- set id_doi = metadata.identifiers|doi_identifier %}
 <dt>{{_('DOI')}}:</dt>
```

Comment:
minor: I would remove the "DOI:" label here. If there is a DOI, "DOI" is displayed below. If there is none, the text says "No minted DOI". In both cases, it's clear the entity is a DOI.

Reply:
done